app/gui2/src/components/ComponentBrowser.vue

```diff
@@ -37,42 +36,24 @@ const graphStore = useGraphStore()
 const props = defineProps<{
   nodePosition: Vec2
   navigator: ReturnType<typeof useNavigator>
-  initialContent: string
-  initialCaretPosition: ContentRange
-  sourcePort: Opt<ExprId>
+  usage: Usage
 }>()
 
 const emit = defineEmits<{
   accepted: [searcherExpression: string, requiredImports: RequiredImport[]]
-  closed: [searcherExpression: string]
-  canceled: []
+  closed: [searcherExpression: string, requiredImports: RequiredImport[]]
+  canceled: [searcherExpression: string, requiredImports: RequiredImport[]]
```

Comment:
Why does `canceled` also report `searcherExpression` and `requiredImports`? Is there any use for that?

Reply:
Well, mostly because it can? (And unused parameters are discarded anyway). But I can remove them.